Resources/Themes/Frontend/BiancoTheme/frontend/_public/src/less/_components/fonts.less

```diff
@@ -1,4 +1,16 @@
-@OpenSansPath: "../../fonts";
+@OpenSansPath: "../../../../../../../../../../../../themes/Frontend/Responsive/frontend/_public/vendors/fonts/open-sans-fontface";
```

Comment:
Bei mir kommt ein Fehler, weil ein '../' fehlt. 

Hast du das Theme als Theme integriert oder als Plugin?

Reply:
ich habe es als Plugin installiert

Reply:
hm ok. Ich habe die fonts einfach in unser Theme kopiert. Sollte so auch erstmal funktionieren.